scripts/utils/upload_and_generate_geojson.py

```diff
@@ -105,22 +94,22 @@
     # This dataset has quite some dimensions, so if we would parse all information the end-user
     # would be overwhelmed by all options. So for the stac items that we generate for the frontend
     # visualizations a subset of the data is selected. Of course, this operation is dataset specific.
-    for k, v in MAP_SELECTION_DIMS.items():
-        if k in ds.dims and ds.coords:
-            ds = ds.sel({k: v})
-        else:
-            try:
-                # assume that coordinates with strings always have same dim name but with n
-                ds = ds.sel({"n" + k: k == v})
-            except:
-                raise ValueError(f"Cannot find {k}")
-
-    dimvals = get_dimension_values(ds, dimensions_to_ignore=DIMENSIONS_TO_IGNORE)
-    dimcombs = get_dimension_dot_product(dimvals)
+    # for k, v in MAP_SELECTION_DIMS.items():
+    #     if k in ds.dims and ds.coords:
+    #         ds = ds.sel({k: v})
+    #     else:
+    #         try:
+    #             # assume that coordinates with strings always have same dim name but with n
+    #             ds = ds.sel({"n" + k: k == v})
+    #         except:
+    #             raise ValueError(f"Cannot find {k}")
+
+    # dimvals = get_dimension_values(ds, dimensions_to_ignore=DIMENSIONS_TO_IGNORE)
+    # dimcombs = get_dimension_dot_product(dimvals)
 
     for var in VARIABLES:
         collection = get_geojson(
-            ds, variable=var, dimension_combinations=dimcombs, stations_dim="stations"
```

Comment:
Why did you change stations to locs? Is this not stations in the CF compliant file?

Reply:
No in the CF-compliant file, it is named: locs.

Reply:
I do think we agreed on naming it stations to allow the F/E to work without any hard-coded if/else statements.. Can you revise the CF compliant file then?